README.md

```diff
@@ -73,3 +73,28 @@ The architecture is as follows:
 - Each destination is expected to be a PDA that then distributes the yield to the climate project
   according to their own rules. For example, the Offset Bridge bridges the funds to Polygon, buys
   NCT etc.
+
+## Usage (latest version -- Yield Router)
+There is a state account owned by the yield controller program which stores the following data:
+- `sunrise_state`: public key of the input yield account which holds funds to be sent to PDAs that will transfer fund to various climate projects (e.g. the offset bridge)
+- `update_authority`: public key of the account having the authority to update the state account
+- `output_yield_accounts`: a vector holding public keys of the PDAs that will transfer funds to various climate project
+- `spend_proportions`: a vector holding the proportions of funds to be sent to each elements in `output_yield_accounts`, the proportions must sum up to 100
+- `spend_threshold`: minimum amount of fund existing in `sunrise_state` account before transfer of funds to `output_yield_accounts` is allowed
+- `total_spent`: total amount of fund sent to various climate projects
+
+This state account needs to be first registered on chain, this only needs to be performed once for a yield controller program. To do so, one can use the `register_state` instruction by calling the `packages/yield-router/registerState.ts` script as follow:
+```ANCHOR_PROVIDER_URL=[anchor_provide_url] STATE_ADDRESS=[state_address] OUTPUT_YIELD_ADDRESS=[output_yield_address] yarn ts-node packages/yield-router/registerState.ts```
```

Comment:
I think it is more common to use curly braces { } or angle brackets < > instead of square brackets [ ] to indicate variables, e.g. ANCHOR_PROVIDER_URL=<anchor_provider_url>

There is a typo for all "anchor_provideR_url"